src/DbSelect.php

```diff
@@ -20,32 +22,11 @@ class DbSelect implements AdapterInterface
 {
     public const ROW_COUNT_COLUMN_NAME = 'C';
 
-    /** @var Sql */
-    protected $sql;
-
-    /**
-     * Database query
-     *
-     * @var Select
-     */
-    protected $select;
-
-    /**
-     * Database count query
-     *
-     * @var Select|null
-     */
-    protected $countSelect;
-
-    /** @var ResultSet */
-    protected $resultSetPrototype;
-
-    /**
-     * Total item count
-     *
-     * @var int
-     */
-    protected $rowCount;
+    protected Sql $sql;
```

Comment:
This is a BC break because the current class is not final and the properties are _only_ protected.

Reply:
Removerd